sdccc/src/main/java/com/draeger/medical/sdccc/sdcri/testclient/TestClientImpl.java

```diff
@@ -359,17 +401,37 @@ void deviceFound(final ProbedDeviceFoundMessage message) {
         }
 
         sdcRemoteDevice.registerWatchdogObserver(this);
+        isConnected.set(true);
+    }
+
+    @Override
+    public void shouldReconnect(final Boolean shouldReconnect) {
```

Comment:
Calling this with false as a parameter may lead to log messages from before to not be affected by the respective whitelist as they should, because of them potentially being processed in the appender, which is using the whitelist being set, too late e.g. after this returns.